platform-hub-web/src/app/help/faq/faq-entries.html

```diff
@@ -53,82 +87,105 @@
           </li>
           <li>
-            Onboard and offboard team members on GitHub (using the <em>Actions</em> button next to a team member, on the project page). <strong>Note:</strong> you can only do this for users that have connected up their GitHub identity within the hub.
+            Onboard and offboard team members on Github (using the <em>Actions</em> button next to a team member, on the project page). <strong>Note:</strong> you can only do this for users that have connected up their Github identity within the hub.
+          </li>
+          <li>
+            Manage the <em>services</em> that are being developed in your project (via the Services tab on your project page).
+
+            <ul>
+              <li>
+                Project services are a way of grouping and breaking up your project into smaller sub-projects / deliverables / components / apps etc. Most resources provided by the platform are scoped to, and associated with, a particular service in your project (e.g. Kubernetes robot tokens).
+              </li>
+            </ul>
+          </li>
+          <li>
+            Manage Kubernetes user tokens for team members.
+          </li>
+          <li>
+            Manage Kubernetes robot tokens for CI builds and deploys (on the project service page that you can drill down to from the services list of your project).
+          </li>
+          <li>
+            View the monthly costs for your project (via the "Bills" tab on the project page).
           </li>
         </ul>
       </div>
     </md-card-content>
   </md-card>
 
-  <md-card ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
+  <md-card id="why-cant-I-create-projects-etc" ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
     <md-card-title>
       <md-card-title-text>
-        <span class="md-title">Why can't I create new projects or perform certain actions?</span>
+        <span class="md-title" layout="row">
+          Why can't I create new projects or perform certain actions on the hub?
+          <span flex></span>
+          <a class="md-icon-button" ui-sref="{'#':'why-cant-I-create-projects-etc'}" target="_self"><md-icon ng-bind="$ctrl.linkIcon"></md-icon></a>
+        </span>
       </md-card-title-text>
     </md-card-title>
     <md-card-content>
       <div class="md-body-1">
         <p>
-          You need to be granted specific permissions to be able to do certain things on the hub
+          You need to be granted specific permissions to be able to do certain things on the hub:
         </p>
         <ul>
           <li>
-            <strong>Project admins</strong> can manage a project's team and onboard/offboard team members
+            <strong>Project admins</strong> can manage a project's team, onboard/offboard team members, manage services of that project and manage Kubernetes user and robot tokens for that project
```

Comment:
Maybe rephrase a little, duplication "of/for that project"

Reply:
Agreed, will rephrase.